tests/setup.ts

```diff
@@ -1,10 +1,25 @@
 import { expect, afterEach } from 'vitest';
 import { cleanup } from '@testing-library/react';
 import matchers from '@testing-library/jest-dom/matchers';
+import { vi } from 'vitest';
 
 // extends Vitest's expect method with methods from react-testing-library
 expect.extend(matchers);
 
+Object.defineProperty(window, 'matchMedia', {
```

Comment:
Just out of curiosity, why it is needed?

Reply:
!! This part was not done by me, it came with the recovery of the Pagination component

Reply:
OKs, I will review it